packages/drivers/file-driver/package.json

```diff
@@ -11,16 +11,50 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
-	"type": "commonjs",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
```

Comment:
Are we trimming these in a separate PR?

Reply:
Yeah.  When we trim, we have to convert all downstream dependencies to node16, so we're doing that in a separate step.